### PR TITLE
Update role-based-access-control.md

### PR DESCRIPTION
**Pull request by juliafmorgado**
### Describe your changes
Updated the definition of Role-Based Access Control (RBAC) to be more precise and easier to understand.

### Related issue number or link (ex: `resolves #issue-number`)
#2436

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by jihoon-seo** [comment]
@juliafmorgado Thank you for your work, but please keep [semantic line breaks](https://sembr.org). It will make review & localization processes easier.

**Review by iamNoah1 — APPROVED** [review]
Technically no changes, so ok from my side.

File: content/en/role-based-access-control.md [hunk 1]
```diff
@@ -2,25 +2,25 @@
...
 ## Problem it addresses
 
-Individually managing access permissions of multiple users across various system resources and data can be complex.
-After all, each user likely needs access to a different set of resources. 
-Let's say an organization has 500 developers (users) and 300 resources with varying permission levels (read, write, and no access). The administrator must ensure each user has the correct permissions for all 300 resources.  
-RBAC simplifies the process by providing predefined access control based on role groups.
+RBAC addresses the challenge of controlling the resources team members and applications can access, 
+as well as the actions they can perform, especially as the number of applications and team members increases. 
+Administrators must ensure that each user has the correct permissions for the resources they need to access, 
+a task that can become cumbersome and error-prone without a structured access control mechanism.
```
Comment (nate-double-u) [inline content/en/role-based-access-control.md:19]: nit (this doesn't need to change in order to get merged in):
```suggestion
Administrators must ensure that each user has the correct permissions for the resources they need to access.
This task can become cumbersome and error-prone without a structured access control mechanism.
```

**Review by nate-double-u — APPROVED** [review]
Thanks for this update @juliafmorgado

/lgtm

**Pull request merged**